app/src/main/java/dev/andrylat/raqimbek/bankingutils/MortgageCalculator.java

```diff
@@ -0,0 +1,29 @@
+package dev.andrylat.raqimbek.bankingutils;
+
+import lombok.NonNull;
+
+import java.util.List;
+
+public class MortgageCalculator {
+  @NonNull
+  public Long calculateMonthlyMortgagePayment(@NonNull List<String> inputList) {
+    var borrowedAmount = Double.valueOf(inputList.get(0));
+    var annualInterestRate = Double.valueOf(inputList.get(1));
+    var numberOfYears = Double.valueOf(inputList.get(2));
+    annualInterestRate /= 100;
+
+    var payBackForEveryCurrencyUnit =
```

Comment:
When you work with Money, it is better to use "BigDecimal" everywhere because calculation will be better.